karma.conf.js

```diff
@@ -0,0 +1,36 @@
+const wpConf = require("./webpack.config")
+wpConf.devtool = "inline-source-map"
+
+module.exports = config => {
+    let findTests = () => {
+        // THIS is why I hate NodeJS. That tool is so immature that you always
```

Comment:
Can I keep it if I put my name?
!|](https://user-images.githubusercontent.com/22097904/29003576-65207056-7ab9-11e7-8c56-b9d1f3dc91db.jpg)